plugin/pkg/scheduler/algorithm/predicates/utils.go

```diff
@@ -89,3 +91,24 @@ func GetEquivalencePod(pod *v1.Pod) interface{} {
 type EquivalencePod struct {
 	ControllerRef metav1.OwnerReference
 }
+
+type hostPortInfo struct {
+	protocol string
+	hostIP   string
+	hostPort string
+}
```

Comment:
The `hostPortInfo` looks much cleaner, thanks

Reply:
You are far too kind, Now I will move to e2e test